src/adapters/surrealDB/types/base.ts

```diff
@@ -0,0 +1,86 @@
+import type { BaseResponse } from '../../../types';
+
+export type SurrealDbResponse = {} & BaseResponse;
+
+export type EnrichedBqlQueryRelation = {
+	$thingType: 'relation';
```

Comment:
only 
$thing
$thingType
$as
$fields
$excludedFIelds
$id

Are real user facing metadatas, the rest should be removed if possible or stored as Symbols (we can do this later tho)

Reply:
sure let's do this later, as it will impact typedb adaptor as well